services/document-manager/backend/requirements.txt

```diff
@@ -20,7 +20,7 @@ requests==2.32.3
 SQLAlchemy==2.0.19
 uwsgi==2.0.23
 uwsgitop==0.11
-Werkzeug==2.3.6
+Werkzeug==3.0.3
```

Comment:
Build was failing in MS Cypress run due to Werkzeug version!

Upgraded to version on core-api and nris-api/backend! Checked out the PR where it was upgraded on core-api and based on that I'm hoping there's no other changes I'm missing: https://github.com/bcgov/mds/commit/2643944d725abd8e8c179a64ea2d856b63d93ad2 😅

Reply:
Ohh I seem to remember there was a reason for why this wasn't upgraded in Docman. Want to take it out of this PR and I'll see if I can dig it up 😅

Reply:
Actually it seems to be running fine, so should be OK to leave in (hopefully)

Reply:
Alrighty! If it blows up test as soon as it's deployed (or doc upload/downloads down't work) we can do up a quick PR to revert.